packages/techdocs-common/src/stages/publish/local.ts

```diff
@@ -107,34 +103,25 @@ export class LocalPublish implements PublisherBase {
     });
   }
 
-  fetchTechDocsMetadata(entityName: EntityName): Promise<TechDocsMetadata> {
-    return new Promise((resolve, reject) => {
-      this.discovery.getBaseUrl('techdocs').then(techdocsApiUrl => {
-        const storageUrl = new URL(
-          new URL(this.config.getString('techdocs.storageUrl')).pathname,
-          techdocsApiUrl,
-        ).toString();
+  async fetchTechDocsMetadata(
+    entityName: EntityName,
+  ): Promise<TechDocsMetadata> {
+    const metadataPath = path.join(
+      staticDocsDir,
+      entityName.namespace,
+      entityName.kind,
+      entityName.name,
```

Comment:
no lowercasing still? :)

Reply:
Yep, should probably create an issue to make this case-insensitive here (in directory names) and URLs as well.

Reply:
I think the directory is quite hidden and an implementation detail. So, it should be fine to keep it case sensitive.

However, it makes sense to make `kind` and `namespace` lowercase in URLs.